packages/components/src/particles/Banner/index.tsx

```diff
@@ -1,21 +1,46 @@
-import { Show, JSXElement } from 'solid-js';
+import { createSignal, JSXElement, onMount, Show } from 'solid-js';
 import Icon from '../Icon';
 
 export type BannerProps = {
   children: JSXElement;
   status: 'success' | 'info';
   withoutIcon?: boolean;
+  closable?: boolean;
+  name?: string;
 };
 
 export default function Banner(props: BannerProps) {
+  const [isVisible, setIsVisible] = createSignal(true);
+
+  onMount(() => {
+    const storedState = localStorage.getItem(`banner-${props.name}`);
```

Comment:
this seems kinda wild?

But if we're going to do it, I'd make it something like `photon-banner-`

Reply:
updated